README.md

```diff
@@ -42,6 +42,11 @@ The `azure.keyvault` plugin resolves credentials or parameters against secrets i
     vault = "myvault"
     ```
 1. [Create a key vault][keyvault] and set the vault name in the config with name of the vault.
+- If you want to specifiy the full url for your vault, use vault-url instead:
+```toml
+   [secrets.config]
+    vault-url = "https://myvault.vault.azure.net"
+```
```

Comment:
The bullet point renders a bit oddly, so how about we make this a new paragraph within the list item instead (along with a spelling fix and a note about which value takes precedence):

```suggestion
1. [Create a key vault][keyvault] and set the vault name in the config with name of the vault.
   
   If you want to specify the full url for your vault, use vault-url instead. Note that `vault-url` takes precedence over the `vault` setting.
```toml
   [secrets.config]
    vault-url = "https://myvault.vault.azure.net"
```
```